portfolio/src/main/java/com/google/sps/servlets/DeletionServlet.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.io.IOUtils;
+
+/** Servlet that deletes a comment whose id is passed to it. */
+@WebServlet("/delete-comment")
+public class DeletionServlet extends HttpServlet {
+
+  /**
+    * POST handler for the /delete-comment route
+    */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
```

Comment:
How is these error reflected to the client side?

Reply:
At the moment that failure is ignored. A future PR might introduce a change to this so that the client would retry

Reply:
The client won't ensure the request is fulfilled. If the request fails, the client will ignore it.

Reply:
since this PR only touches the backend, I likely won't change this behaviour within this PR, but i can push another bugfix PR dealing with the issue